### PR TITLE
ENH: Add team info and grant number

### PR DESCRIPTION
**Pull request by bpaniagua**

### CODE REVIEW

**Comment by bpaniagua** [comment]
@BardiyaAk @jcfr please review and merge. THanks!

File: team.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+title: Team
+permalink: /team/
+---
+
+# Team
+
+SAM development is a collaboration between Brown University and Rhode Island Hospital, Kitware, Cleveland Clinic, Mayo Clinic and Queens University.
+
+## Brown University and Rhode Island Hospital
+<a href="https://www.brown.edu/academics/medical/about/departments/orthopaedics/bioengineering/" target="_blank">Assistive and Restorative Technology Laboratory</a> <br></p>
```
Comment (amymmorton) [inline team.md:12]: incorrect Brown lab : change to      Bioengineering Laboratory

File: team.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+<img style="float: left; margin: 0 20px 20px 0;" src="../images/Kitware.png" alt="Kitware Medical Computing Group">
+<a href="https://www.kitware.com/teams/medical-computing" target="_blank">Medical Computing Group</a> <br></p>
+
+
+## Cleveland Clinic
+<img style="float: left; margin: 0 20px 20px 0;" src="../images/cleveland.jpg" alt="Cleveland Clinic Lerner Research Institute">
+<b>Lerner Research Institute</b> <br></p>
+
+## Mayo Clinic
+<img style="float: left; margin: 0 20px 20px 0;" src="../images/Queens.png" alt="Mayo Clinic  Assistive and Restorative Technology Laboratory">
```
Comment (amymmorton) [inline team.md:25]: Queens.png - correct to Mayo image?

**Review by BardiyaAk — COMMENTED** [review]
do we need a brown.png logo too?

**Comment by bpaniagua** [comment]
Yes! I did not find it anywhere in the website.

**Comment by jcfr** [comment]
The branch has been rebased and preview of proposed changes is now available clicking on any of these links:

| Deploy Preview link in `netlify` bot commend above. See [here](https://github.com/BrownBiomechanics/autoscoperm.slicer.org/pull/1#issuecomment-1114380820) |
|--|
| image|

or 

| `Details` link in pull request checks |
|--|
| image |

**Comment by bpaniagua** [comment]
@BardiyaAk do we have a logo yet?

**Comment by jcfr** [comment]
@amymmorton @BardiyaAk @bpaniagua This is now ready for review and integration

**Review by jcfr — APPROVED** [review]

**Comment by jcfr** [comment]
| Preview: https://deploy-preview-1--autoscoperm-slicer-org-preview.netlify.app/team/ | 
|--|
|image |

**Comment by jcfr** [comment]
| Preview: https://deploy-preview-1--autoscoperm-slicer-org-preview.netlify.app/team/ | 
|--|
| image |

**Review by BardiyaAk — COMMENTED** [review]
all changes look good to me, This is the Brown's logo (https://i.pinimg.com/originals/ec/b4/de/ecb4ded29f113845c6a8ba7e807f9ebb.png)

**Comment by jcfr** [comment]
| Preview: https://deploy-preview-1--autoscoperm-slicer-org-preview.netlify.app/team/ | 
|--|
| image |

cc: @amymmorton @BardiyaAk @bpaniagua

**Comment by bpaniagua** [comment]
This is great, thanks JC, please merge.

**Pull request merged**